src/styles/html.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for a browser to read your web document.
Same for font styles. Set `font-family` to the html tag and will be inherited throughout the page. There are exceptions but you can just inherit it where needed